projects/legacy/components/primitive-textfield/primitive-textfield.component.ts

```diff
@@ -128,7 +128,7 @@ export class TuiPrimitiveTextfieldComponent
 
         const {nativeElement} = this.focusableElement;
 
-        return (nativeElement.previousElementSibling ||
+        return (nativeElement.parentElement?.firstElementChild ||
```

Comment:
It breaks the following case:

* https://taiga-previews--pr9641-input-time-projectio-74ix7oht.web.app/components/select#native-select
* https://taiga-ui.dev/components/select#native-select